_component_design/tooltip.md

```diff
@@ -17,35 +17,22 @@ helix-ui-javascript: true
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
-
-### When to use
-
-Use a tooltip when you need to provide in-context help information. (for example, when describing what will happen when a workflow is initiated). A tooltip can also be used to reveal hidden information, provide extra or supplemental information to users, clarify uses when features could be misinterpreted, or explain an item on a page when it does not have a text label.
-
-### When not to use
-
-Do not use a tooltip if users must repeatedly refer to a tooltip to understand an option or to complete a workflow. We recommend you do not use a tooltip when users needs to interact with the tip's content, or when users integrate with the site on a mobile device.
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Best practice
+## When to use
 
-- Tooltips are perfect for short sentences and sentence fragments that assist the user in their understanding.
-- The ideal tooltip is not obvious, nor does it repeat content that is already visible on the site.
-- Place a tooltip in a subtle location, and ensure it does not interfere with the user or obscure any objects of interest. - Use the tooltip on icon images, text, and table rows.
-- The maximum width for the tooltip is 400px.
-- Ensure there are no interactive controls inside the tooltip.
+Use a tooltip in the following scenarios.
 
-{% endcolumn %}
+- You want to provide in-context help information to the user. For example, use a tooltip to describe what happens when a user initiates a workflow.
+- To reveal hidden information, or provide extra or supplemental information to users.
+- To clarify uses when features could be misinterpreted, or explain an item on a page when it does not have a text label.
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+Do not use a tooltip in the following scenarios.
 
-{% figure [caption:"Meet the tooltip pattern"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/components/help/tooltips/tooltip-hero.svg"/>
-{% endfigure %}
+- Do not use a tooltip if users must repeatedly refer to a tooltip to understand an option or to complete a workflow.
+- Do not use a tooltip when users need to interact with the tooltip content, or when users are working on a mobile device.
```

Comment:
Feel like this needs more clarification. I assume by tooltip content that you mean "content inside a tooltip". That being said, what do in contexts when I do want to interact with the content in a tooltip? Do I use a modal? Is there even an alternative.

Reply:
Also, I would separate these two clauses into separate bullets since they are different use cases. 

----

- When users need to interact with the tooltip content.
- When users are working on a mobile device.

----